src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.resx

```diff
@@ -700,7 +700,7 @@
     <value>10</value>
   </data>
   <data name="LicenseUrlWarning.Text" xml:space="preserve">
-    <value>PackageLicenseURL property was detected, but it has been deprecated. Please change to either an Expression or File.</value>
+    <value>PackageLicenseURL detected but is deprecated. Please change to either an Expression or File.</value>
```

Comment:
This may be more consistent with the other string:

> PackageLicenseURL detected but is deprecated. Use Expression or File instead.

Or update the other message to be more polite :)

Reply:
Changed both to make them consistent and polite 👍